ui/lib/waypoint-client/ServerServiceClientPb.ts

```diff
@@ -4,7 +4,11 @@
  * @public
  */
 
-// GENERATED CODE -- DO NOT EDIT!
+// Code generated by protoc-gen-grpc-web. DO NOT EDIT.
+// versions:
+// 	protoc-gen-grpc-web v1.4.2
+// 	protoc              v3.17.3
+// source: pkg/server/proto/server.proto
```

Comment:
Didn’t expect this to change. Maybe I have a newer version of `protoc-gen-grpc-web` now.